db/migrate/20210216132553_create_grades.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGrades < ActiveRecord::Migration[6.1]
+  def change
+    create_table :grades do |t|
+      t.integer :grade
```

Comment:
it should be renamed, `name`, `number`, etc...
think about this

Reply:
renamed columns and put limit 1.

Reply:
Renamed from grade to number.